src/lib/CLValue/Option.ts

```diff
@@ -144,6 +144,10 @@ export class CLOption<T extends CLValue> extends CLValue {
     return this.data;
   }
 
+  toJSON(): any {
+    return this.isNone() ? 'None' : this.data.unwrap().toJSON();
```

Comment:
Would it be better to return `null` rather than 'None'? Seems closer in meaning to the true sense of None